.luacheckrc

```diff
@@ -5,6 +5,7 @@ ignore = {
 	"131", -- Unused global variable
 	"431", -- Shadowing an upvalue
 	"432", -- Shadowing an upvalue argument
+	"581", -- Negation of a relational operator
```

Comment:
I assume this is to have Luacheck not throw warnings on unit tests that presumably explicitly check lt/gt etc. metamethods, but why disable this for all of builtin?

Reply:
`vector.dot(forward, up) < 0.000001` returns false if the left side is NaN, but `vector.dot(forward, up) >= 0.000001` also returns false if the left side is NaN. On the other hand, `not (vector.dot(forward, up) < 0.000001)` returns true if the left side is NaN. This is used in a check that was added. IMO the warning should be disabled everywhere because it is a bad warning--it raises false positives when it comes to NaN.